data.sql

```diff
@@ -4,4 +4,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
     VALUES ('Agumon', '2020-02-03', 0, true, 10.23),
            ('Gabumon', '2018-11-15', 2, true, 8),
            ('Pikachu', '2021-01-07', 1, false, 15.04),
-           ('Devimon', '2017-05-12', 5, true, 11);
+           ('Devimon', '2017-05-12', 5, true, 11),
+           ('Charmander', '2020-02-08', 0, false, 11),
+           ('Plantmon', '2021-11-15', 2, true, 5.7),
+           ('Squirtle', '1993-04-02', 3, false, 12.13),
+           ('Angemon', '2005-06-12', 1, true, 45),
+           ('Boarmon', '2005-06-07', 7, true, 20.4),
```

Comment:
- [ ] @Fene-87 great work on your project, however, you have changed the weight from a negative figure to a positive, which will affect the queries, kindly change the values and also update the 3 queries dealing with weight.